src/commands/ssh.ts

```diff
@@ -94,15 +94,15 @@ const sshAction = async (args: yargs.ArgumentsCamelCase<SshCommandArgs>) => {
     throw "Server did not return a request id. Please contact support@p0.dev for assistance.";
   }
 
-  const { request } = result;
+  const { request, privateKey, linuxUserName } = result;
 
   await sshOrScp(
     authn,
-    requestToSsh(request),
+    requestToSsh(request, { gcloud: { linuxUserName } }),
```

Comment:
🤔 it's weird that there's a mention of "gcloud" in the ssh base command. I'd expect anything GCP-related (and it's GCP, not `gcloud`, correct?) to be in the `plugins/google` directory.

Reply:
Good point, I will move this.
`gcloud` is the constant we have been using for better or worse. True, it is the name of the CLI.
On the other hand, Google Cloud Platform was renamed to just Google Cloud some time ago, so that invalidates the GCP abbreviation? (Of course people still use it.)

Reply:
Actually, on second thought I think this is not that weird. I wanted to make it clear that the linuxUserName will only be used for gcloud in the called function.
I can hide this field inside PluginRequest to make it less confusing.

Reply:
I defined a new type `CliRequest` to differentiate from `PluginRequest` that we get back from the p0 backend. The `CliRequest` type has an additional field `cliLocalData` which contains the `linuxUserName` for gcp. It's undefined for aws.